src/Sets/EmptySet.jl

```diff
@@ -340,8 +340,36 @@ Translate (i.e., shift) an empty set by a given vector.
 ### Output
 
 The empty set.
+
+### Notes
+
+See also [`translate!(::EmptySet, ::AbstractVector)`](@ref) for the in-place version.
 """
 function translate(∅::EmptySet, v::AbstractVector)
+    return translate!(∅, v)
```

Comment:
obs: in this case, and also for the universe, i decided not to copy

Reply:
I agree. But I think the documentation should not be the generic one as for the other types. Referring to `translate!` as the in-place version of `translate` does not make much sense to me here because they are identical.

Reply:
ok. here `translate!` is only included for completeness with respect to other methods, but it's really a no-op so i just remove the docstring altogether..

Reply:
You removed it from `translate` for `Universe` instead (hence the build failure).